handbook/rbac/rbac.md

```diff
@@ -0,0 +1,128 @@
+
+<figure>
+    <img src="../../img/rbac-concepts.png" width="500" />
+    <figcaption>Role-based access control concepts in SystemLink</figcaption>
+</figure>
+
+### Workspace
+
+A workspace is a complete encapsulation of all systems, data, and other resources within SystemLink. If a user is not a member of a given workspace they cannot access or modify any of the resources within the workspace. This is true for access via the SystemLink web application and the SystemLink REST API. The SystemLink Advanced Server license is required to create multiple workspaces, but there is no limit on the number of workspaces that can be created. If a user is a member of multiple workspaces resource from each workspace will be shown together within the grids, dashboards, and other data views throughout SystemLink.
+
+<figure>
+    <img src="../../img/rbac-block-diagram.png" width="500" />
+    <figcaption>Two workspaces with unique resources in each workspace. The the resources users can access is determined by their workspace membership. The actions a user can take on each resource is determined by their role.</figcaption>
+</figure>
+
+Systems, data, analysis routines (collectively referred to as resources) are unique to individual workspaces. No resource occupies multiple workspaces. Some resources may be moved between workspaces. If a resource is duplicated the user is prompted to chose a workspace for the new resource.
+
+!!! note "Special Considerations for DataFinder"
+
+    DataFinder instances are global (meaning any user can view the DataFinder instance), but DataFinder Search Areas defined within a DataFinder instance are scoped to a single workspace. Search areas can only index File Ingestion Service data for the DataFinder Search Area's corresponding workspace.
+
+### Archived Workspaces
+
+Workspaces may be archived if they are no longer in use. Archival does not delete resources or move resources to a new database or file storage location. Archived workspaces are only accessible to users with the [**Server Administrator**](#server-administrator-role) role. Resources in archived workspaces cannot be created, updated, or deleted. Resources may be duplicated or moved into a non-archived workspace. All active alarms are automatically cleared when a workspace is archived.
+
+### Users and Roles
+
+Within each workspace user, systems, and analysis procedures are assigned roles. Role grant privileges that allow access to the various SystemLink plugins and the data exposed by those plugins. Privileges explicitly grant what you *can* do - there are no *deny* privileges. Due to this a user may be assigned multiple roles within a single workspace that collectively describe what the user has access to. This is useful to prevent the proliferation of multiple different roles that have similar privileges. Instead simple roles may be defined and be composed together by assigning a user each role.
```

Comment:
The first sentence reads weird. Not sure what change needs to be made - maybe @jjaco2 can weigh in.

```suggestion
Within each workspace, users, systems, and analysis procedures are assigned roles. Roles grant privileges that allow access to the various SystemLink plugins and the data exposed by those plugins. Privileges explicitly grant what you *can* do - there are no *deny* privileges. Due to this a user may be assigned multiple roles within a single workspace that collectively describe what the user has access to. This is useful to prevent the proliferation of multiple different roles that have similar privileges. Instead, simple roles may be defined and composed together by assigning a user each role.
```

Reply:
Refactored the first sentence and included other suggestions.